packages/bigtable/src/table.js

```diff
@@ -659,7 +677,17 @@ Table.prototype.getRows = function(options, callback) {
  *   }
  * ];
  *
- * table.insert(entries, function(err, apiResponse) {});
+ * table.insert(entries, callback);
+ *
+ * //-
+ * // A stream mode is also provided, this is useful when making a large number
```

Comment:
This is more of an FYI than anything else...

HBase provides an [interesting approach](https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/client/BufferedMutator.html) to streams that the [Java Cloud Bigtable leverages.](https://github.com/GoogleCloudPlatform/cloud-bigtable-client/blob/master/bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java).  The basic idea is to bundle a finite number of Mutations in a 1-2MB request behind the scene.  The Java implementation tackles retries of individual failures, which is a non-trivial problem.

Under load, Bigtable can produce various types of transient failures (connections timeout, a handful of mutations in the batch timeout).  Some failures are recoverable, some are not.